### ISSUE TITLE
Suggestion: support symmetric handler for FromNillable

### ISSUE THREAD

**Issue by gwolves** [moznion/go-optional#25]
I often map some structs with pointers (intended to express optional values) into Option types.

Example:
```go
type A {
    X *int
    Y *string
}

type B {
    X Option[int]
    Y Option[string]
}
```

I can easily map pointer typed value into Option values:
```go
var a A
...
b := B{
    X: optional.FromNillable(a.X),
    Y: optional.FromNillable(b.Y),
}
```

However, in the opposite case, I have to add some boilerplate code:
```go
var b B
...
var x *int
B.X.IfSome(func(value int) {
    x = &value
})

var y *int
B.Y.IfSome(func(value string) {
    y = &value
})

a := A{X: x, Y: y}
```

I believe this process could be simplified with a symmetric handler for FromNillable. (i.e. ToNillable)
```go
func (o Option[T]) ToNillable() *T

var b B
...
a := A{
    X: b.X.ToNillable(),
    Y: b.Y.ToNillable(),
}
```

Thank you for your hard work on this library!

**Comment by moznion** [comment]
Hi @gwolves, thank you for your suggestion!
I just implemented `UnwrapAsPtr()` method at #28 so I'll be happy if this would work for you.

**Comment by moznion** [comment]
I published [v0.11.0](https://github.com/moznion/go-optional/releases/tag/v0.11.0) which contains this change.
